### PR TITLE
Use cSpell for spell check

### PR DESCRIPTION
**Pull request by taiki-e**

### CODE REVIEW

File: .github/.cspell/dictionary.txt [hunk 1]
```diff
@@ -144,33 +130,29 @@ readline
 README
 recv
 Reloadable
+repr
+repr
```
Comment (OTL) [inline .github/.cspell/dictionary.txt:134]: Could you try uniq command to detect other possible duplications?

**Review by OTL — COMMENTED** [review]
I think it's OK but it will lose GitHub integration, right?

**Comment by taiki-e** [comment]
Yes, but this allows us to share the configuration file with the vs code extension, which I consider more useful than GHA action that sometimes breaks with odd errors.

**Review by OTL — APPROVED** [review]
Thanks!

**Pull request merged**